.github/workflows/ci.yml

```diff
@@ -143,7 +143,7 @@ jobs:
           - ubuntu-latest
           - [self-hosted, linux, arm64]
           - macos-latest
```

Comment:
```suggestion
          - macos-12-large
```

Reply:
Shouldn't it be `macos-13-large`, actually?

Reply:
yes, I fixed

Reply:
No, we want 12 actually. See my other comment.

Reply:
I'll rollback to 12 because the issue mentioned before

Reply:
We should use the same version for both our CI and our release build. Now we are using `macos-latest` for our CI but `macos-13-large` for our release?